### ISSUE TITLE
Keyboard user can't add works to collections or add child works

### ISSUE THREAD

**Issue by chrisdaaz** [samvera/hyrax#3453]
### Descriptive summary

Hyrax Master 3.0.0beta

### Rationale

Keyboard users should be able to act on buttons in the UI that allow works to be added to collections or child works to be added to parent works.

### Expected behavior

The `Add` buttons on the Relationships tab of the Edit Work page should be accessible via keyboard alone.

### Actual behavior

When tabbing through the page, the tabs skip over the `Add` buttons.

![image](https://user-images.githubusercontent.com/24395592/49252501-d158fe00-f3e9-11e8-996e-2fa6ec885ee9.png)

### Steps to reproduce the behavior

1. Visit the Edit work page of a work
2. navigate to the relationships section
3. attempt to add the work to a collection or add a child work via the keyboard

### Related work

Link to related tickets or prior related work here.

**Comment by no-reply** [comment]
We need someone to test this against a recent 2.x release.

If this is broken in 2.x, a backport of any fix would be highly desirable

**Comment by chrisdaaz** [comment]
just tested this on [nurax-stable](https://nurax-stable.curationexperts.com) and it's the same issue, so i think this broken in 2.x

**Comment by dlim87** [comment]
QA - this is not working as of:

Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 17d42738a88955afb8ae6e278f37e51163505ee1

Nurax Version
ENV: production
SHA: 673e2225fe0fbb611ff74513f230bd0b63f091f1
Branch: 2019-09-17-10-00
Updated: 17 Sep 2019 15:00:28

**Comment by Dananji** [comment]
@dlim87 If you could please specify the browser in which this did not work, I think that would be helpful.
Thanks!

**Comment by dlim87** [comment]
Gave it another look on nurax-dev
<details><summary>nurax information</summary>QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 1a8ac5f2e1ffb5c61af599b539253e075ffd50fd 
Nurax Version
ENV: production
SHA: 48af94f8289fe2a63fb2fa4aa484bf95f3530e5e
Branch: 2019-09-29-10-00
Updated: 29 Sep 2019 15:00:17</details>

It DOES work on windows: chrome, firefox edge. Mac: chrome
it does NOT work on mac: firefox, safari

**Comment by dlim87** [comment]
And firefox on mac is a similar situation. It requires turning on "all controls" in system settings > keyboard > shortcuts, and then adding `accessibility.tabfocus` = 7 in `about:config`.

This is super dumb, but apparently it's a OSX level thing that needs to be configured by each client. So this can probably be closed

**Comment by Dananji** [comment]
Yes :+1:

**Comment by kellyChex** [comment]
Closing as this is a user issue.
